app/controllers/registrations_controller.rb

```diff
@@ -46,6 +46,20 @@ def needs_password?(user, params)
       params[:user][:password].present?
   end
 
+  def unsubscribe
+    if user_signed_in?
+      success = SendGridList.newsletter_unsubscribe(current_user.email)
+      if success
+        flash[:notice] = 'You have been unsubscribed from the newsletter.'
+      else
+        flash[:alert] = 'An error occurred unsubscribing you from the newsletter. Please email us at ankit@teawithstrangers.com.'
```

Comment:
@ankitshah811 Review copy: error unsubscribing

This may happen if SendGrid's API rate limits us, or something else happens that fails to unsubscribe the user. This error will be shown in that case.